### DISCUSSION TITLE
Ghost Setup on INAV 4.10

### DISCUSSION THREAD

**Discussion by Signellox** [iNavFlight/inav#8170] [Q&A] [answered]
Hello,
I tried to conect the gost with the Ghost protocol to MATEK  411 Wing . But I can not move the servos with my Horus. 
Moving the Flight Cotroller -> Servo move
My settings are:
-   GHST protocol 
- Serial Out Ghost to TX1
-  UART1  Serial RX on
- Receiver type : Serial 
- Serial Receiver Provider: GHST
- Prearming Checks all green
- INAV Configurater 4.10 Firmware 4.10

**Comment by DzikuVx** [comment, marked as answer]
As all my craft run Ghost, I can confirm that it's working just fine

**Reply by Signellox** [reply]
Thank you, I have the problem and spended until now 6 hours to solve it. Servos did not move until now.

**Reply by Signellox** [reply]
![image](https://user-images.githubusercontent.com/35845416/176108945-a233ce17-6170-4212-a149-f66aad595f83.png)

**Reply by Signellox** [reply]
![image](https://user-images.githubusercontent.com/35845416/176109129-0e0cc6fb-3d70-4afd-958a-038119ccb63b.png)

**Reply by Signellox** [reply]
Problem solved  after another 3 hours. Installing everything new again.
